src/node_platform.cc

```diff
@@ -24,6 +24,44 @@ static void BackgroundRunner(void* data) {
   }
 }
 
+BackgroundTaskRunner::BackgroundTaskRunner(int thread_pool_size) {
+  for (int i = 0; i < thread_pool_size; i++) {
+    std::unique_ptr<uv_thread_t> t { new uv_thread_t() };
+    if (uv_thread_create(t.get(), BackgroundRunner, &background_tasks_) != 0)
+      break;
+    threads_.push_back(std::move(t));
+  }
+}
+
+void BackgroundTaskRunner::PostTask(std::unique_ptr<Task> task) {
+  background_tasks_.Push(std::move(task));
+}
+
+void BackgroundTaskRunner::PostIdleTask(std::unique_ptr<v8::IdleTask> task) {
+  CHECK(false && "idle tasks not enabled");
```

Comment:
`UNREACHABLE()`?  Doesn't print a message although that could be added.

Reply:
done!